docs/zh/kustomization.yaml

```diff
@@ -0,0 +1,287 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ----------------------------------------------------
+# Example kustomization.yaml content.
+#
+# This file declares the customization provided by
+# the kustomize program.
+#
+# Since customization is, by definition, _custom_,
+# there are no sensible default values for the fields
+# in this file.
+#
+# The field values used below are merely examples, not
+# to be copied literally.  The values won't work if
+# they happen to be references to external files that
+# don't exist.
+#
+# In practice, fields with no value should simply be
+# omitted from kustomization.yaml to reduce the content
+# visible in configuration reviews.
+# ----------------------------------------------------
+# Kustomization 的 apiVersion 和 kind 
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# 为所有 resources 添加 namespace
+namespace: my-namespace
+
+# 该字段的值将添加在所有资源的名称之前
+# 例如 将资源名称 “wordpress” 变为 “alices-wordpress” 
+namePrefix: alices-
+
+# 该字段的值将添加在所有资源的名称后面
+# 例如 将资源名称 “wordpress” 变为 “wordpress-v2”
+# 如果资源类型为 ConfigMap 或 Secret ，则在哈希值之前附加后缀
+nameSuffix: -v2
+
+# 为所有资源和 selectors 增加 Labels
+commonLabels:
+  someName: someValue
+  owner: alice
+  app: bingo
+
+# 和 Labels 一样， 增加 Annotations
+# 为 key:value 键值对
+commonAnnotations:
+  oncallPager: 800-555-1212
+
+# 此列表中的每条记录都必须是一个存在的 YAML 资源描述文件
+# 这些 YAML 资源描述文件将被 kustomize 读取，修改和发布
+# 资源由文档标记（"---"）分隔
```

Comment:
I'm not quite familiar with the Chinese names for k8s resources.
By reading the paragraph, I suggest to change it to something like this for easy
readability.
```
# 此列表中的每条记录都必须是一个存在的 YAML 资源描述文件
# 一个YAML 资源描述文件可以含有多个由（“---”）分隔的资源。
# kustomize 将读取这些YAML文件中的资源，对其进行修改并
# 发布在kustomize的输出中。
```